sinon/index.d.ts

```diff
@@ -1,6 +1,6 @@
-// Type definitions for Sinon 1.16.0
+// Type definitions for Sinon 2.1.0
 // Project: http://sinonjs.org/
-// Definitions by: William Sears <https://github.com/mrbigdog2u>
+// Definitions by: William Sears <https://github.com/mrbigdog2u>, Jonathan Little <https://github.com/rationull>
```

Comment:
At this point @mrbigdog2u hasn't showed up after 2 days, so I would appreciate the help if you don't mind. 

I'll wait another day or so before merging to see if he does want to review it, though.